ee/clickhouse/views/test/test_clickhouse_funnel_correlation.py

```diff
@@ -322,19 +431,87 @@ def test_correlation_endpoint_with_properties(self):
+                    "success_people_url": ANY,
+                    "failure_people_url": ANY,
                     # "odds_ratio": 1 / 121,
                     "correlation_type": "failure",
                 },
             ],
         )
 
+    def test_properties_correlation_endpoint_provides_people_drill_down_urls(self):
+        """
+        Here we are setting up three users, two with a specified property but
+        differing values, and one with this property absent. We expect to be
+        able to use the correlation people drill down urls to retrieve the
+        associated people for each.
+        """
+
+        with freeze_time("2020-01-01"):
+            self.client.force_login(self.user)
+
+            update_or_create_person(distinct_ids=["Person 1"], team_id=self.team.pk, properties={"$browser": "1"})
+            update_or_create_person(distinct_ids=["Person 2"], team_id=self.team.pk, properties={"$browser": "1"})
+
+            events = {
+                "Person 1": [
+                    # Failure / $browser::1
+                    {"event": "signup", "timestamp": datetime(2020, 1, 1)},
+                ],
+                "Person 2": [
+                    # Success / $browser::1
+                    {"event": "signup", "timestamp": datetime(2020, 1, 1)},
+                    {"event": "view insights", "timestamp": datetime(2020, 1, 3)},
+                ],
+                "Person 3": [
+                    # Success / $browser not set
+                    {"event": "signup", "timestamp": datetime(2020, 1, 1)},
+                    {"event": "view insights", "timestamp": datetime(2020, 1, 3)},
+                ],
+            }
+
+            create_events(events_by_person=events, team=self.team)
+
+            odds = get_funnel_correlation_ok(
+                client=self.client,
+                team_id=self.team.pk,
+                request=FunnelCorrelationRequest(
+                    events=json.dumps([EventPattern(id="signup"), EventPattern(id="view insights")]),
+                    date_to="2020-04-04",
+                    funnel_correlation_type=FunnelCorrelationType.PROPERTIES,
+                    funnel_correlation_names=json.dumps(["$browser"]),
+                ),
+            )
+
+            (browser_correlation,) = [
+                correlation
+                for correlation in odds["result"]["events"]
+                if correlation["event"]["event"] == "$browser::1"
+            ]
+
+            (notset_correlation,) = [
+                correlation for correlation in odds["result"]["events"] if correlation["event"]["event"] == "$browser::"
+            ]
+
+            assert get_people_for_correlation_ok(client=self.client, correlation=browser_correlation) == {
+                "success": ["Person 2"],
+                "failure": ["Person 1"],
+            }
+
+            assert get_people_for_correlation_ok(client=self.client, correlation=notset_correlation) == {
```

Comment:
For reference and consistency, do you recommend using this type of assert rather than the class function self.assertEqual? Usage wise they're both clear to me but I think we should start syncing them together so tests aren't written differently in different places